test/users.js

```diff
@@ -31,6 +31,7 @@ describe('Users', function() {
     user.username = 'JaneD';
     user.password = '123password';
     user.email = 'jane@aol.com';
+    user.role = 'admin';
```

Comment:
Maybe add a test or tests showing other user roles can't insert or edit users. A big CYA there.

Reply:
Agreed will do that.